API-Samples/utils/util_init.cpp

```diff
@@ -415,7 +427,9 @@ void destroy_debug_report_callback(struct sample_info &info) {
 }
 
 void init_connection(struct sample_info &info) {
-#ifndef _WIN32
+#ifdef __ANDROID__
+    // Do nothing on Android.
+#elif defined(_WIN32)
```

Comment:
His objection was at the point that 2 files (LICENSE.txt and LICENSE) are vague to distinguish and  now it looks better.
I would withdraw the change for an instance and will update if he still has a suggestion.